### DISCUSSION TITLE
Colors change when enable go extension

### DISCUSSION THREAD

**Discussion by mirusky** [golang/vscode-go#2354] [Help] [answered]
I need help, I can't program with the new colors, I'm color blind and theses changed affected my productivity.

Before:
![image](https://user-images.githubusercontent.com/54698990/179747533-e3fb931d-a198-470c-821c-a4e5450e28f9.png)

After:
![image](https://user-images.githubusercontent.com/54698990/179747607-3733388c-3542-4ce2-b102-21c90952c159.png)

**Comment by hyangah** [comment, marked as answer]
It's a regression in v0.35.0. We just released v0.35.1 with a fix.
Sorry for the trouble.

https://github.com/golang/vscode-go/issues/2355
